### ISSUE TITLE
hubscr test failures

### ISSUE THREAD

**Issue by jtrmal** [usnistgov/SCTK#5]
BTW, there is also an issue with hubscr.pl (I checked it's a problem even for perl v5.20.2):
```
make[1]: Entering directory '/mnt/work/home/jtrmal/SCTK/src/hubscr'
Testing hubscr.pl
./RunTests.pl
   Running test 'test1-sastt', operation 'test', options '-G -f rttm -F rttm -a', directory 'test1-sastt.test'
      Executing command
      Comparing output
[ERROR] Test test1-sastt has failed.  Diff output is :
diff -i -x CVS -x .DS_Store -x log -x '*lur' -I '[cC]reation[ _]date' -I md-eval -r test1-sastt.test/sastt-case1.sys.rttm.filt.alignments/segmentgroup-132.html test1-sastt.base/sastt-case1.sys.rttm.filt.alignments/segmentgroup-132.html
 61d60
 < jg.drawStringRect("fe016",0, 197, scale*1500, "left");
 62a62
 > jg.drawStringRect("fe016",0, 197, scale*1500, "left");
 69a70,71
 > jg.drawLine(scale*600, 85, scale*600, 55);
 > jg.drawLine(scale*600, 55, scale*600, 25);
 73,75d74
 < jg.drawLine(scale*800, 55, scale*800, 25);
 < jg.drawLine(scale*600, 85, scale*600, 55);
 < jg.drawLine(scale*600, 55, scale*600, 25);
 77a77
 > jg.drawLine(scale*800, 55, scale*800, 25);
```
(the diff continues)

The test tries to compare the files
`test1-sastt.ref/sastt-case1.sys.rttm.filt.alignments/segmentgroup-132.html`
`test1-sastt.ref/sastt-case1.sys.rttm.filt.alignments/segmentgroup-133.html`
vs
`test1-sastt.test/sastt-case1.sys.rttm.filt.alignments/segmentgroup-132.html`
`test1-sastt.test/sastt-case1.sys.rttm.filt.alignments/segmentgroup-133.html`

The html renders in browser look pixel-identical

when I sort the lines and run diff again, most of the differences disappear (132.html) or there are only a couple of draw line primitives are different (133.html) -- seems like most of the differences are only caused by a different ordering of the draw primitives or the lines are drawn in an opposite direction:
```
$ diff <(sort test1-sastt.test/sastt-case1.sys.rttm.filt.alignments/segmentgroup-132.html) <(sort test1-sastt.base/sastt-case1.sys.rttm.filt.alignments/segmentgroup-132.html)
249,250c249,250
< jg.drawLine(scale*600, 175, scale*600, 205);
< jg.drawLine(scale*600, 235, scale*600, 175);
---
> jg.drawLine(scale*600, 175, scale*600, 235);
> jg.drawLine(scale*600, 235, scale*600, 205);
254c254
< jg.drawLine(scale*700, 205, scale*700, 235);
---
> jg.drawLine(scale*700, 175, scale*700, 205);
259c259
< jg.drawLine(scale*800, 175, scale*800, 205);
---
> jg.drawLine(scale*800, 205, scale*800, 175);
265c265
< jg.drawLine(scale*900, 205, scale*900, 235);
---
> jg.drawLine(scale*900, 235, scale*900, 175);
```

**Comment by jtrmal** [comment]
I bisected it down to rev b907b91521a0056540ddcc60f67ef1a52f2b0836 which is actually first revision in which the files 
```
test1-sastt.base/sastt-case1.sys.rttm.filt.alignments/segmentgroup-132.html
test1-sastt.base/sastt-case1.sys.rttm.filt.alignments/segmentgroup-133.html
```
exist, but it still failed, so it seems the test either never worked as intended or the perl has changed something substantial (as the commit dates back to 2009), but not sure what

**Comment by jfiscus** [comment]
It’s perl.  The tests passed during the Babel program.

**Comment by jfiscus** [comment]
I'm going to comment out the SASTT Tests for now to get a clean make check.  to my knowledge, that code wasn't used after RT09 so this code isn't  high priority.

**Comment by jfiscus** [comment]
I made a commit (d1e4744669e62be18cfaff97622bd0d2a4301ab5) to try to fix this.  I'm concerned it may not work on another system.

**Comment by jtrmal** [comment]
Thanks Jon -- I did some more fixes, see #6 and now the test passes on two perls on MacOsX (5.18 and 5.26).
EDIT: 6.26->5.26

**Comment by jtrmal** [comment]
BTW, albeit it might not look like, I didn't do these changes blindly, I just noticed some other parts of the files have different (non-alphabetic ordering), so I just traced these things down to where they were outputted.
